src/main/resources/db-master-changelog.xml

```diff
@@ -1407,7 +1407,7 @@
         <customChange class="no.ndla.taxonomy.migration.CleanUpDuplicateResourcesSqlChange"/>
     </changeSet>
 
-    <changeSet id="20230823 Add schema and extension" author="Gunnar Velle" runAlways="true" >
+    <changeSet id="20230823 Add schema and extension" author="Gunnar Velle">
```

Comment:
Sikker på at du vil ta bort denne?

Reply:
det var egentlig bare for testing så nei jeg er ikke sikker :smile: 
Egentlig syns jeg det er litt uheldig å ha den, men om den har løst et problem så har ikke jeg gjort noe testing :grimacing:

Reply:
Meiner det er veldig fint å ha om du setter opp ny postgres lokalt og så migrerer databaser fra test, for ellers får du ikkje extension-schemaet i basen sjølv om changelogen seier du har det.